model/lib/email/presentation_email.dart

```diff
@@ -1,17 +1,86 @@
 
 import 'package:equatable/equatable.dart';
+import 'package:intl/intl.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/core/unsigned_int.dart';
+import 'package:jmap_dart_client/jmap/core/utc_date.dart';
+import 'package:jmap_dart_client/jmap/mail/email/email.dart';
+import 'package:jmap_dart_client/jmap/mail/email/email_address.dart';
+import 'package:jmap_dart_client/jmap/mail/email/keyword_identifier.dart';
+import 'package:model/mailbox/select_mode.dart';
+import 'package:model/extensions/email_address_extension.dart';
 
-class PresentationThread with EquatableMixin {
+class PresentationEmail with EquatableMixin {
```

Comment:
It seems you only use it to display in the list mails (in ThreadView)?

Reply:
ThreadView => MailboxView ?

Reply:
It shows up on the email screen too.

Reply:
![129555644-229abd19-d1db-4d31-bfe1-3f1f7e59804c](https://user-images.githubusercontent.com/6462404/129865574-a20a0b7a-0539-45bb-9686-a13f966c2e0f.png)

Currently

Reply:
> It shows up on the email screen too.

please take care, when it has the body and attachment also

Reply:
Please say what you want.